### PR TITLE
Provide translation of readme file in IEP-608 (IEP-754)

### PR DESCRIPTION
**Pull request by NatashaLi-ESPRESSIF**
## Description

Provide translation for the missing part in readme_cn against readme.

### CODE REVIEW

**Comment by NatashaLi-ESPRESSIF** [comment]
@leeebo, pls help with the review.

File: README_CN.md [hunk 1]
```diff
@@ -444,15 +495,197 @@ IDF Eclipse 插件可支持不同语言。如需更改，请按照以下步骤
...
+注意，右侧的 `Start parse` 按钮默认将处于禁用状态。此时，可以点击窗口右下方的 `开始` 按钮，这将生成一个 dump 文件，并同时解禁 `Start parse` 按钮。
+
+此后，点击 `Start parse` 按钮，从 Eclipse 控制台查看解析脚本的输出：
+![](docs/images/AppLvlTracing_5.png)
+
+<a name ="updateEspIdfMaster"></a>
+
+# ESP-IDF Master 更新
+
+您如何在使用 ESP-IDF 的 master 分支，并希望进行升级：请打开 Eclipse 插件，前往 `乐鑫` -> `ESP-IDF 工具管理器`，并点击 `更新 ESP-IDF master`。
```
Comment (leeebo) [inline README_CN.md:631]: 如何 -> 如果

File: README_CN.md [hunk 2]
```diff
@@ -444,15 +495,197 @@ IDF Eclipse 插件可支持不同语言。如需更改，请按照以下步骤
...
+	mvn clean verify -Djarsigner.skip=true
+	```
+
+以上命令将生成 p2 更新站点 artifact：
+* 名称：`com.espressif.idf.update-*`
+* 存储路径：`releng/com.espressif.idf.update/target`
+
+后续可按照 <a href="https://github.com/espressif/idf-eclipse-plugin#installPluginsUsingLocalFile">介绍</a> 进行安装。
+
+# 如果获得最新的开发构建
```
Comment (leeebo) [inline README_CN.md:658]: 如果 -> 如何

**Review by leeebo — COMMENTED** [review]
LGTM!

**Comment by NatashaLi-ESPRESSIF** [comment]
@alirana01 , pls review and merge!

**Review by alirana01 — APPROVED** [review]
LGTM!
**Will add screenshots with updated language in the main branch for IEP-608.**

**Pull request merged**